doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Common NFVI Telco Taskforce
+
+## Table of Contents
+* [1. Overview](#1.0)
+* [2. Scope](#2.0)
+* [3. Principles](#3.0)
+* [4. Technology Direction](#4.0)
+  * [4.1 IO Virtualisation](#4.1)
+  * [4.2 SmartNICs](#4.2)
+  * [4.3 Acceleration Cards](#4.3)
+  * [4.4 FPGA](#4.4)
+  * [4.5 GPUs/NPUs](#4.5)
+  * [4.6 EPA/NFD](#4.6)
+* [5. Available Specifications](#5.0)
+
+<a name="1.0"></a>
+## 1. Overview
+
+Initially organized early in 2019, the Common Network Function Virtualisation Infrastructure Telecom Taskforce (CNTT) was initially created in response to rapid changes in how networking applications are being designed, built and managed, plus a growing recognition of a perceived functional gap between the previous standard infrastructure models and the architectures needed to support Network Function Virtualization (NFV) applications.  Organizationally the Common Telco Network Function Virtualisation Infrastructure (NFVI)  project, jointly hosted by GSMA and the Linux Foundation, operates as an open committee responsible for creating and documenting an industry aligned Common NFVI Framework.  The CNTT group was created with the intent that it would create the NFVI framework, and eventually morph into an on-going project under the auspices of the GSMA and the Linux Foundation umbrellas.  The final on-going operational form of the group will be determined as the project evolves. 
```

Comment:
"The CNTT group was created .." since CNTT is a taskforce, I think, "The CNTT was created ..." works

Reply:
@pgoyal01 done! please make as resolved if satisfied by changes made